src/controllers/settings_controller.js

```diff
@@ -135,6 +136,11 @@ SettingsController.post('/subscribe', async (request, response) => {
 SettingsController.post('/skiptunnel', async (request, response) => {
   try {
     await Settings.set('notunnel', true);
+
+    if (Platform.getOS() === 'linux-openwrt') {
+      RouterSetupApp.onSetupComplete();
```

Comment:
should this go here? or perhaps we should call onSetupComplete at the beginning of startHttpsGateway and startHttpGateway, so that no matter how the gateway gets started we complete setup (if started). The onSetupComplete would just do nothing if setup mode weren't entered.

Reply:
We can't do it inside `startHttpGateway`, because that would kill the secondary captive portal before the user makes it there.